_posts/2019-04-01-vuzix-blade.md

```diff
@@ -103,23 +113,27 @@ It's like someone is following you from behind with a projector and is projectin
 Hold a smartwatch right in front of you in a readable position.
 It's kinda like that, but transparent and without losing the functionality of one of your arms.
 
-So instead of describing it to people, we just put it on their face and they just are immediately captivated by what they're experiencing.
-It takes a moment to learn how to switch your eyes's focus on the heads-up display and back to your surroundings.
-Once you master, this it becomes very natural to interact with the display.
-Seeing that transparent interface is something a lot of people really have to take time to wrap their heads around.
+
+So instead of describing it to people, we just put it on their face and they are just immediately captivated by what they're experiencing.
+It takes a moment to learn how to switch your eyes's focus on the heads up display and back to your surroundings.
+Once you master this it becomes very natural to interact with the display, however staring at it for prolonged periods is not what's it's meant for.
+Seeing that transparent interface for the first time is something a lot of people really have to take time to wrap their heads around the idea.
```

Comment:
A lot of people have difficulty in wrapping their heads around the idea of that transparent interface when trying out the glasses for the first time.